aaad/authorizers.py

```diff
@@ -174,10 +174,14 @@ def filter_response(self, resource, data, actas):
                 return ""
             return framework.filterResponseBody(data, allowed_namespaces, resource)
 
-        def _parse_permissions_file(self, filename):
-            with open(filename, 'r') as data_file:
-                return yaml.load(data_file)
-            return ''
+        def _parse_permissions_files(self, filenames):
+            permissions = {}
+            for item in filenames.split(','):
+                filename = item.strip(' ,') # strip ',' and ' '
```

Comment:
You only need `.strip()` since the commas will already be gone.  Also `strip()` strips on all whitespace not just space.